src/Microsoft.AspNetCore.Authentication.OpenIdConnect/OpenIdConnectHandler.cs

```diff
@@ -320,6 +320,14 @@ protected virtual Task<bool> HandleSignOutCallbackAsync()
                 Scope = string.Join(" ", Options.Scope)
             };
 
+            string maxAge = OpenIdConnectHelper.GetMaxAgeParameterFromTimeSpan(Options.MaxAge);
+            // Add the 'max_age' parameter to the authentication request if MaxAge is at least one second.
```

Comment:
`max_age=0` is not disallowed by the specification and can be seen as an equivalent to `prompt=login` (in OpenID 2.0 PAPE, `max_auth_age=0` was explicitly allowed and served this exact purpose).

That would be great to avoid making the ASP.NET Core implementation stricter than the specification, specially since this won't be trivial to debug for beginners (invalid values are silently ignored).